pkg/api/latest/latest.go

```diff
@@ -21,7 +21,20 @@ var OldestVersion = unversioned.GroupVersion{Group: "", Version: "v1"}
 // may be assumed to be most preferred to least preferred, and clients may
 // choose to prefer the earlier items in the list over the latter items when presented
 // with a set of versions to choose.
-var Versions = []unversioned.GroupVersion{{Group: "", Version: "v1"}}
+var Versions = []unversioned.GroupVersion{
```

Comment:
I don't even remember what this is used for.  I thought we used reflection to find out if it was an openshift type.